packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -399,7 +399,7 @@ class _MaterialScrollbarState extends RawScrollbarState<_MaterialScrollbar> {
   void handleHover(PointerHoverEvent event) {
     super.handleHover(event);
     // Check if the position of the pointer falls over the painted scrollbar
-    if (isPointerOverScrollbar(event.position, event.kind)) {
+    if (isPointerOverScrollbar(event.position, event.kind, forHover: true)) {
```

Comment:
I don't love `forHover`, but I needed a way to distinguish different mouse events - click/drag v. hover

Reply:
It's not horrible.  I mean, that's what it's for.